verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -579,7 +579,8 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
                 <MaterialTable
                     tableRef={this.tableRef}
                     columns={[
-                        ...(this.props.user.roles.includes('curator')
+                        ...(this.props.user.roles.includes('curator') ||
```

Comment:
This would also give admins the option to edit a row (which would cause an error if they clicked on it). It's probably best to not give admins visibility to this three-dot menu. They can still delete a single row by clicking on the selection box and clicking delete in the toolbar.

Reply:
ok reverted